### PR TITLE
Regression files updated

### PR DESCRIPTION
**Pull request by i-Alex**
Text hex format is used instead of binary.

### CODE REVIEW

File: sdk/src/test/java/com/horizen/box/CertifierRightBoxSerializerTest.java [hunk 1]
```diff
@@ -20,8 +20,15 @@
     @Before
     public void setUp() {
         Pair<byte[], byte[]> keyPair = Ed25519.createKeyPair("12345".getBytes());
-        // Note: current box bytes are also stored in "src/test/resources/certifierrightbox_bytes"
+        // Note: current box bytes are also stored in "src/test/resources/certifierrightbox_hex"
         box = new CertifierRightBox(new PublicKey25519Proposition(keyPair.getValue()), 1000, 20, 10);
+
```
Comment (g666) [inline sdk/src/test/java/com/horizen/box/CertifierRightBoxSerializerTest.java]: Comments should be removed.

File: sdk/src/test/java/com/horizen/box/RegularBoxSerializerTest.java [hunk 2]
```diff
@@ -20,8 +20,15 @@
     @Before
     public void setUp() {
         Pair<byte[], byte[]> keyPair = Ed25519.createKeyPair("12345".getBytes());
-        // Note: current box bytes are also stored in "src/test/resources/regularbox_bytes"
+        // Note: current box bytes are also stored in "src/test/resources/regularbox_hex"
         box = new RegularBox(new PublicKey25519Proposition(keyPair.getValue()), 1000, 10);
+
```
Comment (g666) [inline sdk/src/test/java/com/horizen/box/RegularBoxSerializerTest.java]: Comments should be removed.

File: sdk/src/test/java/com/horizen/proposition/PublicKey25519PropositionSerializerTest.java [hunk 3]
```diff
@@ -19,8 +19,15 @@
     @Before
     public void beforeEachTest() {
         Pair<byte[], byte[]> keyPair = Ed25519.createKeyPair("12345".getBytes());
-        // Note: current proposition bytes are also stored in "src/test/resources/publickey25519proposition_bytes"
+        // Note: current proposition bytes are also stored in "src/test/resources/publickey25519proposition_hex"
         proposition = new PublicKey25519Proposition(keyPair.getValue());
+
```
Comment (g666) [inline sdk/src/test/java/com/horizen/proposition/PublicKey25519PropositionSerializerTest.java]: Comments should be removed.

File: sdk/src/test/java/com/horizen/secret/PrivateKey25519SerializerTest.java [hunk 4]
```diff
@@ -16,8 +15,15 @@
 
     @Before
     public void beforeEachTest() {
-        // Note: current secret bytes are also stored in "src/test/resources/privatekey25519_bytes"
+        // Note: current secret bytes are also stored in "src/test/resources/privatekey25519_hex"
         key = PrivateKey25519Creator.getInstance().generateSecret("12345".getBytes());
+
```
Comment (g666) [inline sdk/src/test/java/com/horizen/secret/PrivateKey25519SerializerTest.java]: Comments should be removed.

File: sdk/src/test/scala/com/horizen/SidechainSyncInfoTest.scala [hunk 5]
```diff
@@ -41,9 +41,11 @@ class SidechainSyncInfoTest extends JUnitSuite with SidechainBlockInfoFixture {
       assertEquals("SidechainSyncInfo known block %d is different".format(i), info.knownBlockIds(i), serializedInfoTry.get.knownBlockIds(i))
 
 
-    /*val out = Some(new FileOutputStream("src/test/resources/sidechainsyncinfo_bytes"))
-    out.get.write(bytes)
-    out.get.close()*/
+
```
Comment (g666) [inline sdk/src/test/scala/com/horizen/SidechainSyncInfoTest.scala]: Comments should be removed.

File: sdk/src/test/java/com/horizen/transaction/RegularTransactionSerializerTest.java [hunk 6]
```diff
@@ -41,18 +41,15 @@ public void beforeEachTest() {
         to.add(new Pair<>(pk5.publicImage(), 20L));
         to.add(new Pair<>(pk6.publicImage(), 90L));
 
-        // Note: current transaction bytes are also stored in "src/test/resources/regulartransaction_bytes"
+        // Note: current transaction bytes are also stored in "src/test/resources/regulartransaction_hex"
         transaction = RegularTransaction.create(from, to, fee, timestamp);
 
-        /*
-        //Save transaction to binary file for regression tests.
         try {
```
Comment (i-Alex) [inline sdk/src/test/java/com/horizen/transaction/RegularTransactionSerializerTest.java]: should be commented

File: sdk/src/test/scala/com/horizen/chain/SidechainBlockInfoTest.scala [hunk 7]
```diff
@@ -58,11 +57,12 @@ class SidechainBlockInfoTest extends JUnitSuite with SidechainBlockInfoFixture {
       assertEquals("SidechainBlockInfo reference is different", info.mainchainBlockReferenceHashes(index), references(index))
     assertEquals("SidechainBlockInfo withdrawalEpochInfo is different", info.withdrawalEpochInfo, serializedInfoTry.get.withdrawalEpochInfo)
 
-    /*
-    val out = Some(new FileOutputStream("src/test/resources/sidechainblockinfo_bytes"))
-    out.get.write(bytes)
-    out.get.close()
-*/
+
```
Comment (g666) [inline sdk/src/test/scala/com/horizen/chain/SidechainBlockInfoTest.scala]: Comments should be removed.

**Pull request merged**